pkg/kubelet/dockertools/labels_test.go

```diff
@@ -116,10 +117,12 @@ func TestLabels(t *testing.T) {
 	expected.PodDeletionGracePeriod = pod.DeletionGracePeriodSeconds
 	expected.PodTerminationGracePeriod = pod.Spec.TerminationGracePeriodSeconds
 	expected.PreStopHandler = container.Lifecycle.PreStop
+	// Touch up HTTPHeaders. Defaulting has converted them from nil to an empty array.
+	expected.PreStopHandler.HTTPGet.HTTPHeaders = make([]api.HTTPHeader, 0)
```

Comment:
as above - I don't think this is needed is it?  Maybe this whole file can be reverted